draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -548,12 +565,21 @@ fragment as described in {{lorawan-schc-payload}}.
 * MAX_ACK_REQUESTS: 8
 
 As only 1 tile is used, its size can change for each downlink, and will be
-maximum available MTU.
+maximum available MTU for unicast. For multicast, it has to be the minimum MTU
+of all devices, RECOMMENDED value is 51.
```

Comment:
I disagree: this is exactly the same rule in multicast as for unicast for the MTU. In multicast, the network has to provide each end-device with the correct datarate settings, which determines the MTU. The original sentence is correct for multicast, without addition

Reply:
Ok